dockerfiles/Dockerfile

```diff
@@ -232,7 +145,12 @@ RUN echo "deb http://deb.debian.org/debian bullseye-backports main" >> /etc/apt/
 RUN rm -f /usr/bin/qemu-*-static
 COPY --from=builder /fluent-bit /fluent-bit
 
+## Copy AWS plugins
+COPY --from=amazon/aws-for-fluent-bit:latest /fluent-bit/kinesis.so /fluent-bit/kinesis.so
```

Comment:
I would verify no binary incompatibility here (it'll be fun as it may only trigger on certain calls) because the AWS image is based on 1.9 codebase.

Reply:
From the initial tests I did, mostly around cloudwatch, this is working fine :)

Reply:
Sure, I would add automated regression tests though as it will likely fail at some point.